fluent-package/Rakefile

```
@@ -366,6 +366,22 @@ class UpdateLockfileTask
         end
       end
     end
+
+    namespace :gemfile do
+      desc "Update Gemfile"
+      task :update do
+        require_relative '../lib/update-gems'
+        cd gemfile_dir do
+          options = {
+            lts: false,
```

```suggestion
            lts: !!ENV['LTS'],
```

We can update Gemfile for LTS like `rake gemfile:update LTS=true`